OneSignalSDK/build.gradle

```diff
@@ -18,6 +19,8 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.6.2'
 
+        classpath 'com.google.gms:google-services:4.3.2'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
Super nit picky here but lets use `'` instead of `"` so its consistent with others

Reply:
Also any reason we don't just add '1.3.72' to the end of the `classpath` instead of `ext.kotlin_version = '1.3.72'`

Reply:
I wanted to start cleaning this, I usually use ext to define versions, but is not a must, I can change to make all use this new format or remove it, let me know what you think